ingestion/src/metadata/ingestion/source/dashboard/qlikcloud/models.py

```diff
@@ -20,14 +20,14 @@
 class QlikApp(BaseModel):
     """QlikCloud App model"""
 
-    description: Optional[str]
-    name: Optional[str]
+    description: Optional[str] = None
+    name: Optional[str] = None
     id: str
     app_id: Optional[str] = Field(alias="resourceId", default=None)
```

Comment:
Nitpick, maybe we could keep it as we have on the other places: `Field(None, alias="resourceId")`

Thought this one was missing for a moment xD

Reply:
ye not sure why its different